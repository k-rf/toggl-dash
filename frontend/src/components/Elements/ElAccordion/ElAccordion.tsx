import { Accordion, AccordionProps } from "@mui/material";

type Props = Pick<AccordionProps, "children" | "expanded" | "sx">;

export const ElAccordion = (props: Props) => {
  return <Accordion {...props} disableGutters />;
};

export type { Props as ElAccordionProps };
