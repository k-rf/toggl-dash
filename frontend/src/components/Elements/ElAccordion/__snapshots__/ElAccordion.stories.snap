// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Elements/ElAccordion/ElAccordion > Default 1`] = `
<body>
  <div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded css-1r6ctgc-MuiPaper-root-MuiAccordion-root"
    >
      <div
        aria-expanded="false"
        class="MuiButtonBase-root MuiAccordionSummary-root css-1aqmfld-MuiButtonBase-root-MuiAccordionSummary-root"
        role="button"
        tabindex="0"
      >
        <div
          class="MuiAccordionSummary-content css-1betqn-MuiAccordionSummary-content"
        >
          Summary
        </div>
      </div>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
          >
            <div
              class="MuiAccordion-region"
              role="region"
            >
              <div
                class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
              >
                Details
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
